dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -729,6 +730,42 @@ def test_correct_total_sample_size_and_counts_and_mutability(self):
         self.assertEqual(col_one_len, len(data['NAME']))
         self.assertEqual(col_two_len, len(data['VALUE']))
 
+    def test_save_and_load(self):
+        datapth = "dataprofiler/tests/data/"
+        test_files = ["csv/guns.csv", "csv/iris.csv"]
```

Comment:
as opposed to loading files, we can just create our own mock data inside the function. Then we no longer are dependent on external files which could disappear or change

Reply:
Since the same test data files are being used for other tests, it might be unnecessary to create mock data - it's still a good point though and I could replace these with pandas data frames